micro-rdk/src/common/config_monitor.rs

```diff
@@ -43,7 +69,12 @@ impl<'a> PeriodicAppClientTask for ConfigMonitor<'a> {
                 app_client.clone().get_config(None).await
             {
                 if self.curr_config != *new_config {
-                    self.restart()
+                    if let Some(config) = new_config.config {
+                        if let Err(e) = self.storage.store_robot_configuration(config) {
```

Comment:
did we consider erasing  the config then restarting?

Reply:
We could. That would leave open the possibility that if you can't connect when you come back online, you won't be able to build a robot until you talk to app again. That might be a risk worth taking.

Reply:
I think this could work, its either a empty config or an invalid config if you can't come back online which both wouldn't work. Should I implement this solution?

Reply:
let's make the assumption that network will be available until the next time we poll config after a reboot. The big risk would be an invalid config but that's on the user

Reply:
OK. @allisonschiang please take @npmenard's suggestion to just erase the config instead of storing it here.